src/arcaflow_plugin_sdk/test_schema.py

```diff
@@ -2117,6 +2161,172 @@ def test_one_of_inline(self):
+            InlineInt.__name__: schema.ObjectType(
+                InlineInt,
+                {
+                    "msg": schema.PropertyType(schema.StringType()),
+                    "code": schema.PropertyType(schema.IntType()),
+                    discriminator_field_name: schema.PropertyType(
+                        schema.IntType()
+                    ),
+                },
+            ),
+            InlineInt2.__name__: schema.ObjectType(
+                InlineInt2,
+                {
+                    "msg2": schema.PropertyType(schema.StringType()),
+                    discriminator_field_name: schema.PropertyType(
+                        schema.IntType()
+                    ),
+                },
+            ),
+        }
+
+        defs = schema._JSONSchemaDefs()
+        json_schema = scope._to_jsonschema_fragment(scope, defs)
+        self.assertEqual(
+            {
+                "$defs": {
+                    IntInlineUnion.__name__: {
+                        "type": "object",
+                        "properties": {
+                            "union": {
+                                "oneOf": [
+                                    {
+                                        "$ref": (
+                                            f"#/$defs/{InlineInt.__name__}"
+                                            "_discriminated_int_"
+                                            f"{inline_int_key}"
+                                        )
+                                    },
+                                    {
+                                        "$ref": (
+                                            f"#/$defs/{InlineInt2.__name__}"
+                                            "_discriminated_int_"
+                                            f"{inline_int2_key}"
+                                        )
+                                    },
+                                ]
+                            }
+                        },
+                        "required": ["union"],
+                        "additionalProperties": False,
+                        "dependentRequired": {},
+                    },
+                    InlineInt.__name__: {
+                        "type": "object",
+                        "properties": {
+                            "msg": {"type": "string"},
+                            "code": {"type": "integer"},
+                            discriminator_field_name: {
+                                "type": "integer",
+                                "const": str(inline_int_key),
+                            },
+                        },
+                        "required": [discriminator_field_name, "msg", "code"],
+                        "additionalProperties": False,
+                        "dependentRequired": {},
+                    },
+                    f"{InlineInt.__name__}"
+                    + f"_discriminated_int_{inline_int_key}": {
```

Comment:
`f"{InlineInt.__name__}"` is equal to `InlineInt.__name__`, so just use the latter at line 2259.

Ditto 2287.

Reply:
changed it